src/qz/utils/CommandParser.java

```diff
@@ -0,0 +1,220 @@
+/**
+ * @author Tres Finocchiaro
+ *
+ * Copyright (C) 2019 Tres Finocchiaro, QZ Industries, LLC
+ *
+ * LGPL 2.1 This is free software.  This software and source code are released under
+ * the "LGPL 2.1 License".  A copy of this license should be distributed with
+ * this software. http://www.gnu.org/licenses/lgpl-2.1.html
+ */
+
+package qz.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import qz.common.Constants;
+import qz.common.SecurityInfo;
+import qz.exception.NullCommandException;
+import qz.installer.Installer;
+import qz.installer.TaskKiller;
+import qz.installer.certificate.PropertiesLoader;
+
+import java.io.File;
+import java.util.*;
+
+import static qz.common.Constants.*;
+import static qz.utils.CommandParser.ExitStatus.*;
+
+public class CommandParser {
```

Comment:
Something like ArgumentParser would probably be a better name for what this is doing.

Reply:
Done, renamed to `ArgParser`.